README.md

```diff
@@ -14,7 +14,7 @@ go get -u github.com/heetch/confita
 
 ## Usage
 
-Confita scans a struct for `config` tags and calls all the backends one after another until the key is found.
+Confita scans a struct for `config` tags and calls all the backends one after another (except if the [backend tag](#backend-tag) is specified) until the key is found.
```

Comment:
I don't think we should specify the edge case here. We should let this sentence as is as a general definition and explain exceptions in a separate paragraph as you did below